lib/ES/DocsRepo.pm

```diff
@@ -0,0 +1,94 @@
+package ES::DocsRepo;
+
+use strict;
+use warnings;
+use v5.10;
+
+use Path::Class();
+
+use parent qw( ES::Repo );
+
+#===================================
+sub new {
+#===================================
+    my ( $class, $tracker, $dir ) = @_;
+
+    $dir = Path::Class::dir( $dir );
+    my $self = $class->SUPER::new(
+        name      => 'docs',
+        git_dir   => $dir->subdir( '.git' ),
+        tracker   => $tracker,
+        url       => 'git@github.com:elastic/docs.git',
+        keep_hash => 0,
+    );
+    $self->{dir} = $dir;
+    return $self;
```

Comment:
I think you still need to `bless` the instance before returning it. Something like 
```perl
return bless $self, $class;
```

But can be that it gets blessed from the parent

Reply:
The parent blesses it, yeah.